HISTORY.md

```diff
@@ -1,5 +1,9 @@
 # Version History
 
+## 0.5_preview / 2021-11-12
```

Comment:
we should change this to 0.5.0_preview

Reply:
Seconded

Reply:
Thirded, will change

Reply:
since it was agreed, I knocked out this change when i did the merge conflicts

Reply:
Thank you!